src/features/lightspeed/api.ts

```diff
@@ -86,14 +100,14 @@ export class LightSpeedAPI {
     );
     try {
       this._completionRequestInProgress = true;
+      this._inlineSuggestionFeedbackSent = false;
       const response = await axiosInstance.post(
         LIGHTSPEED_SUGGESTION_COMPLETION_URL,
         inputData,
         {
           timeout: ANSIBLE_LIGHTSPEED_API_TIMEOUT,
         }
       );
-      this._completionRequestInProgress = false;
```

Comment:
Why is this line removed? The return in next line won't run the finally block.

Reply:
Keeing the line will not hurt anything, but I think we'd better take it out because the `finally` block is executed even if `return` is inserted in the middle of the `try` block.  For example, the following code:
```
function foo(i) {
  try {
    if (i > 0) {
      console.log("i > 0");
      return "POSITIVE";
    } else if (i == 0) {
      throw new Error("i == 0");
    } else {
      console.log("i < 0");
    }
    return "NEGATIVE";
  } catch (e) {
     console.log(e.toString());
     return "ZERO";
  } finally {
     console.log("FINALLY");
  }
}

console.log(foo(1));
console.log(foo(0));
console.log(foo(-1));
```
executes like:
```
$ node foo.js
i > 0
FINALLY
POSITIVE
Error: i == 0
FINALLY
ZERO
i < 0
FINALLY
NEGATIVE
```
What do you think?

Reply:
Thanks for the explanation. The changes looks good to me.